apps/hash-graph/lib/graph/src/api/rest/json_schemas/entity_type.json

```diff
@@ -1,6 +1,6 @@
 {
-  "$id": "https://blockprotocol.org/type-system/0.2/schema/meta/entity-type",
-  "description": "Specifies the structure of an Entity Type",
+  "title": "Entity Type",
```

Comment:
`$schema` wasn't there to begin with as it is incompatible with the OpenAPI spec. `$id` was removed (can add again), but needs to not be a URL and just an identifier, otherwise the generator generates garbage.

Reply:
What do you mean by "generates garbage"? Didn't it work before?

Reply:
It works here (as openapi-generator ignores it on the top level), but if `$id` is in any of the sub-definitions they will error out and generate as `object`. To stay consistent I removed it from the top too (although I can add it again)